### ISSUE TITLE
Installation process design

### ISSUE THREAD

**Issue by arkag** [ryanrudolfoba/SteamDeck-Clover-dualboot#7]
Hey there,

Thanks for the work on this. I'm wondering if you could explain a design decision to me. 

Generally, it's considered bad form to manage binary files via git, because git isn't capable of managing diffs correctly when the file changes. This can drastically slow down repository actions as that file changes, because the history isn't very accurate at managing the diff of the file.

It seems like there'd be a way to pull down the clover ISO on install, or pull down a different build type from Clover's own GH repo.

Thanks again, I appreciate your thoughts on this!
Alex

**Comment by ryanrudolfoba** [comment]
Thanks for the feedback. We've chatted on discord about my reason but I'll post it here too in case anyone else is reading -

I've used the ISO because I'm not sure if the Clover guys delete / change the link then the script will fail.
I've also avoided using pacman as that will require read-only to be disabled, and it alters a bunch of stuff like keyring etc etc. With the ISO I just extract it and put the files in the correct location. I've also taken the hash of the ISO to verify its authenticity.

I appreciate your explanation how git releases work. I'll update the script so it pulls the ISO / zip from Clover repos. Thanks!
